shadows/framework/src/main/java/org/robolectric/shadows/ShadowCountDownTimer.java

```diff
@@ -1,60 +1,83 @@
 package org.robolectric.shadows;
 
+import static org.robolectric.util.reflector.Reflector.reflector;
+
 import android.os.CountDownTimer;
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
 import org.robolectric.annotation.RealObject;
 import org.robolectric.shadow.api.Shadow;
 import org.robolectric.util.ReflectionHelpers.ClassParameter;
+import org.robolectric.util.reflector.Accessor;
+import org.robolectric.util.reflector.Direct;
+import org.robolectric.util.reflector.ForType;
 
 @Implements(CountDownTimer.class)
 public class ShadowCountDownTimer {
   private boolean started;
-  private long countDownInterval;
-  private long millisInFuture;
-
-  @RealObject CountDownTimer countDownTimer;
+  @RealObject private CountDownTimer realCountDownTimer;
 
   @Implementation
   protected void __constructor__(long millisInFuture, long countDownInterval) {
```

Comment:
Can we just get rid of this `__constructor__` method? At this point it seems unnecessary.